### ISSUE TITLE
Bug: dark Footer accessibility problem

### ISSUE THREAD

**Issue by techtolentino** [Qiskit/qiskit.org#3175]
### Steps to reproduce the problem

1. Go to the [Qiskit Metal page](https://qiskit.org/metal)
2. Scroll down to the `Footer`
4. See that the footer links are dark colored on a dark background, creating an accessibility issue 

### What is the current behavior?

Links in the dark-themed `Footer` are inaccessible

### What is the expected behavior?

Links should be accessible

### Screenshots

https://user-images.githubusercontent.com/6276074/235714299-9079f2dc-a218-4e56-b722-435a4ae78fd3.mov

### Browser

Chrome/FF

**Comment by techtolentino** [comment]
This doesn't seem to be an issue anymore



![Image](https://github.com/Qiskit/qiskit.org/assets/6276074/2084118a-e653-40d7-b638-6e5d39dba243)
